weave-js/src/components/PagePanelComponents/Home/Browse3/pages/util.tsx

```diff
@@ -32,6 +33,17 @@ export const renderCell = (value: any) => {
   if (typeof value === 'boolean') {
     return value ? 'True' : 'False';
   }
+  if (typeof value === 'number') {
+    return (
+      <Box
+        sx={{
+          textAlign: 'right',
+          width: '100%',
+        }}>
+        {value.toFixed(4)}
```

Comment:
I think we want to skip this if the value is an integer, like `true_count`?

Reply:
i like it - will make this change